SimCalorimetry/EcalSimProducers/src/EcalTimeDigiProducer.cc

```diff
@@ -70,30 +69,30 @@ void EcalTimeDigiProducer::accumulate(PileUpEventPrincipal const &e,
   edm::Handle<std::vector<PCaloHit>> ebHandle;
   e.getByLabel(m_hitsProducerTagEB, ebHandle);
```

Comment:
This one should be changed as well.

Reply:
PileUpEvent only uses label and not token